config/source/300-rabbitmqsource.yaml

```diff
@@ -81,19 +81,10 @@ spec:
               exchangeConfig:
```

Comment:
Is the flattening of the exchange and queue configs into `rmqResources` going to a part of this PR as well?

Reply:
I wrote about that in the issue, lets discuss it there